source/common/http/codes.cc

```diff
@@ -7,124 +7,147 @@
 #include "envoy/stats/scope.h"
 
 #include "common/common/enum_to_int.h"
-#include "common/common/fmt.h"
 #include "common/common/utility.h"
 #include "common/http/headers.h"
 #include "common/http/utility.h"
 
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_join.h"
+
 namespace Envoy {
 namespace Http {
 
-void CodeUtility::chargeBasicResponseStat(Stats::Scope& scope, const std::string& prefix,
-                                          Code response_code) {
+CodeStatsImpl::CodeStatsImpl() {}
```

Comment:
nit: del empty constuctor/destructor

Reply:
added https://github.com/envoyproxy/envoy/issues/5182 for discussing inline dtors.

Reply:
That's fine, I don't really care as long as we are generally consistent. We already do this for mocks for the same reason. I will stop commenting on this.